sdk/core/azure-core/CMakeLists.txt

```diff
@@ -152,9 +152,8 @@ endif()
 
 if (MSVC)
   # Disable warnings:
-  # - C6285: (<non-zero constant> || <non-zero constant>) -> VBProject static analysis on Functional header:
-  #          _Is_large, regression from VS version 19.28.29915.0 to 19.29.30037.0
-  target_compile_options(azure-core PUBLIC /wd6285)
+  # - C6101: Returning uninitialized memory '*Mtu' -> libcurl calling WSAGetIPUserMtu from WS2tcpip.h
+  target_compile_options(azure-core PUBLIC /wd6101)
```

Comment:
Is there a cpp file for which, when being compiled, this warning gets emitted? If so, it is better to disable the warning via "... `ifdef _MSC_VER`, `pragma warning disable`".

If there's a header which includes the 3rd party header file that generates the warning, it also is better to disable as "... `ifdef _MSC_VER`, `pragma warnings push`, `pragma warning disable`, `include <3rd party>`, `ifdef MSC` `pragma pop`"

Reply:
@vhvb1989 Is this warning only coming from libcurl cpp file? Can you share where it is coming from?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2474